app/models/saved_claim/dependency_claim.rb

```diff
@@ -81,23 +81,32 @@ def regional_office
     []
   end
 
-  private
-
-  def partitioned_686_674_params
-    dependent_data = parsed_form
-
-    college_student_data = dependent_data['dependents_application'].extract!(*STUDENT_ATTENDING_COLLEGE_KEYS)
-
-    { college_student_data: college_student_data, dependent_data: dependent_data }
+  # Run after a claim is saved, this processes any files/supporting documents that are present
+  def process_attachments!
+    supporting_documents = parsed_form.dig('dependents_application', 'supporting_documents')
+    if supporting_documents
+      files = PersistentAttachment.where(guid: supporting_documents.map { |doc| doc['confirmation_code'] })
+      files.find_each { |f| f.update(saved_claim_id: id) }
+    end
   end
 
-  def upload_to_vbms(path)
+  def upload_to_vbms(path, doc_type = '148')
```

Comment:
```suggestion
  def upload_to_vbms(path, doc_type: '148')
```

Prefer keyword args if possible - this change might require updating any calls to the `#upload_to_vbms` method

Reply:
@kfrz I've updated this as suggested, thank you :)  It is a helpful suggestion, I prefer this way better too.  I didn't have to update any calls to this method because currently we are just using the default doc type.  The change to look up and pass in different doc types will be coming in a future PR.  Thank you for pointing that out though!